### PR TITLE
VIES by European Commission (Independent Publisher)

### PR DESCRIPTION
**Pull request by tposzytek**
This is my first time I tried writing custom code, but it works! Please, let me know if its ok.
I had to do it, because VIES service works only via SOAP, so I had to create code that forwards request to SOAP and returns results.

---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

- [X] I attest that the connector works and I verified by deploying and testing all the operations. 
- [X] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [X] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [X] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [X] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [X] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [X] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [X] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [X] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md. 

Designer test

Flow test

Console test

### CODE REVIEW

**Comment by sriyen-msft** [comment]
[[certify-connector]]

File: independent-publisher-connectors/VIES by European Commission/script.csx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        var data = JObject.Parse(requestBody);
+        var countryCode = ((string)data["countryCode"]).Trim();
+        var vatNumber = ((string)data["vatNumber"]).Trim().Replace(" ", string.Empty).Replace("-", string.Empty).Replace(".", string.Empty);
+        
+        var url = "http://ec.europa.eu/taxation_customs/vies/services/checkVatService";
+        var xml = @"<s:Envelope xmlns:s='http://schemas.xmlsoap.org/soap/envelope/'><s:Body><checkVat xmlns='urn:ec.europa.eu:taxud:vies:services:checkVat:types'><countryCode>"+countryCode+"</countryCode><vatNumber>"+vatNumber+"</vatNumber></checkVat></s:Body></s:Envelope>";
+
+        try
+        {
+            using (var client = new WebClient())
```
Comment (sriyen-msft) [inline independent-publisher-connectors/VIES by European Commission/script.csx:31]: @tposzytek I came to know this during preparing your connector for production deployment that - The creation of an HTTP client is not allowed in custom code in any form either using HTTPClient or WebClient. While we work to find a safe way to achieve some of the functionality, we ask you to explore alternate ways to achieve similar functionality. For example, for this method, you can create a simple connector for europa.eu and use the output from the connector in this field. Alternatively, you can either create a self-managed functions app to do the same. Thirdly, if you are the owner of the backend API you do similar work in your backend.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by tposzytek** [comment]
Hi, what do you mean by "For example, for this method, you can create a simple connector for europa.eu and use the output from the connector in this field."? I can't find any information about endpoints available at europa.eu site :( and VAT validation is available only via SOAP.
Why weren't there any warnings/ error messages saying that web/http clients are not allowed? Code was validated successfully and all tests went fine.
Unfortunately I am can't build a self managed functions app, as this would mean I will be using my subscription to handle everyone's requests?
Finally, I'm not the owner of the backend API.
Does it mean this connector is going to be rejected?

Best regards,

Tomasz Poszytek

e-mail: ***@***.***
tel: +48 603 993 372
www/ blog: https://poszytek.eu
twitter: @TomaszPoszytek
________________________________

**Comment by sriyen-msft** [comment]
> 

@tposzytek Apologies for the inconvenience, Since it security-related issue which we recently found in the platform, we are actively working on updating all the tools to block Webclient and HttpClient references in the connector artifacts. 
As of now, we can not move forward with your submission, but we can keep your submission on hold and will explore if we find anything alternatives. Thank you so much.

**Comment by tposzytek** [comment]
Hey, thanks! In that case please out it on hold and let me know when there are any alternatives to make secure soap requests. Have a good day!

Best regards,

Tomasz Poszytek

e-mail: ***@***.***
tel: +48 603 993 372
www/ blog: https://poszytek.eu
twitter: @TomaszPoszytek
________________________________

**Comment by Amjed-Ayoub** [comment]
Dear partner,

We apologize for the delay you might be experiencing while we are working on a high volume of connector requests. Rest assured that we are doing everything we can to address this volume. We are taking measures like improving our processes and hiring additional team members to guarantee resolving your connector requests in a timely manner. 

Please let us know if you have any questions in the meantime.

**Comment by Amjed-Ayoub** [comment]
Hello @tposzytek,
I hope you are doing well.
Apologies for the delay in the process, we will move forward with the connector as is.
Congratulations, your connector is approved. We will move forward with the certification process. We are preparing your connector for production deployment and will queue up for the next deployment schedule. Once your connector onboards the next deployment schedule, it starts to deploy your connector in our production environments that typically takes 3-4 weeks. 
Please let us know if you have any questions.
Thank you very much for working with us.

**Comment by Amjed-Ayoub** [comment]
[[certify-connector]]

**Review by sriyen-msft — APPROVED** [review]

**Comment by sriyen-msft** [comment]
@tposzytek Congratulations! The deployment of your connector and documentation page is now complete, We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program, Thanks.

**Pull request merged**